code/game/turfs/simulated/floor/mineral_floor.dm

```diff
@@ -17,8 +17,9 @@
 
 
 /turf/open/floor/mineral/Initialize()
-	broken_states = list("[initial(icon_state)]_dam")
-	. = ..()
+	if(!broken_states)
+		broken_states = list("[initial(icon_state)]_dam")
+		. = ..()
```

Comment:
I'm guessing it was this tbh

Reply:
move this to above the if and unindent it

Reply:
It can't be above the if, further up the chain does a similar thing to set default values for broken_states